### PR TITLE
fix(json-ocsf): add check_id field in json-ocsf output

### PR DESCRIPTION
**Pull request by pedrooot**
### Context

This pr adds new field check_id inside unmapped_data for json-ocsf output.

### Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/lib/outputs/json_ocsf/json_ocsf.py [hunk 1]
```diff
@@ -107,6 +107,7 @@ def fill_json_ocsf(finding_output: FindingOutput) -> DetectionFinding:
             ),
             type_uid=DetectionFindingTypeID.Create,
             type_name=DetectionFindingTypeID.Create.name,
+            unmapped_data={"check_id": finding_output.check_id},
```
Comment (jfagoagas) [inline prowler/lib/outputs/json_ocsf/json_ocsf.py]:
```suggestion
            unmapped={"check_id": finding_output.check_id},
```

Review the format here https://schema.ocsf.io/1.1.0/classes/detection_finding?extensions=

**Review by jfagoagas — CHANGES_REQUESTED** [review]
Please update the tests for the `fill_json_ocsf` to reflect the change.

File: prowler/lib/outputs/json_ocsf/json_ocsf.py (hunk 1 above)
Comment (pedrooot) [reply prowler/lib/outputs/json_ocsf/json_ocsf.py]: Hey! @jfagoagas just fixed py-ocsf-models in this pr: https://github.com/prowler-cloud/py-ocsf-models/pull/32 once it is updated I will apply the change. 
Thanks for the catch!

**Review by jfagoagas — APPROVED** [review]
👏

**Pull request merged**